### ISSUE TITLE
Problem with Vue 2.7 and hasInjectionContext

### ISSUE THREAD

**Issue by wJoenn** [unjs/unhead#209]
### Environment

Vue 2.7.14, that's all you should need.

### Reproduction

Make a 2.7.14 vite app and install @unhead/vue

### Describe the bug

I'm pretty sure `hasInjectionContext` required Vue >= 3.3 so any app using an older version will get this error
Shouldn't you update the peer dependencies ?

### Logs

```sh
✘ [ERROR] No matching export in "node_modules/vue/dist/vue.runtime.esm.js" for import "hasInjectionContext"

    node_modules/@unhead/vue/dist/shared/vue.e88efe84.mjs:1:35:
      1 │ import { version, unref, nextTick, hasInjectionContext, inject, ref, watchEffect, watch, getCurrentInstance, onBeforeUnmount, onDeactivated, onActivated } from 'vue';
```
```

**Comment by harlan-zw** [comment]
Thanks for the issue, this is fixed in 1.5.0.

**Comment by wJoenn** [comment]
Cheers for the speed mate 🚀
